targets/targets.json

```diff
@@ -7878,7 +7880,7 @@
         "inherits": ["Target"],
         "macros": ["MBED_MPU_CUSTOM"],
         "default_toolchain": "GCC_ARM",
-        "supported_toolchains": ["GCC_ARM", "IAR", "ARM"],
```

Comment:
All Cypress targets are failing ARMC6 compilation. This needs investigation from partners, so we are currently binding Cypress targets to ARMC5.

Reply:
Tracking issue https://github.com/ARMmbed/mbed-os/issues/9830